test/test_nn_quantized.py

```diff
@@ -33,7 +34,7 @@ def test_relu_api(self):
         self.assertEqual(qY, qY_hat)
 
 
-class ModuleAPITest(TestCase):
+class ModuleAPITest(QuantizationTestCase):
```

Comment:
We should probably move the scriptability tests into their own test class. Maybe into `ScriptabilityTests`.